.editorconfig

```diff
@@ -1,14 +1,14 @@
-; http://editorconfig.org
-
-root = true
-
-[*]
-indent_style = space
-indent_size = 2
-end_of_line = lf
-charset = utf-8
-trim_trailing_whitespace = true
-insert_final_newline = true
-
-[*.md]
+; http://editorconfig.org

```

Comment:
Some of this diff is line ending garbage since I wrote a lot of this on my Windows machine. Annoying.